src/frontends/paddle/src/op/unique.cpp

```diff
@@ -0,0 +1,41 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "default_opset.hpp"
+#include "openvino/frontend/paddle/node_context.hpp"
+#include "openvino/opsets/opset10.hpp"
+
+namespace ov {
+namespace frontend {
+namespace paddle {
+namespace op {
+NamedOutputs unique(const NodeContext& node) {
+    auto x = node.get_input("X");
+
+    std::vector<Output<Node>> outputs;
+    NamedOutputs named_outputs;
+
+    auto axis = node.get_attribute<std::vector<int32_t>>("axis");
+
+    auto dtype_str = node.get_attribute<std::string>("dtype");
+    auto dtype = dtype_str == "int32" ? element::i32 : element::i64;
+
+    if (axis.size() != 0) {
+        auto axis_node = std::make_shared<default_opset::Constant>(element::i32, Shape{}, axis);
+        outputs = std::make_shared<ov::opset10::Unique>(x, axis_node, true, dtype, dtype)->outputs();
```

Comment:
Can we use default_opset other than opset10 ?

Reply:
To the best of my knowledge, the unique operator is defined in opset10

- https://docs.openvino.ai/2023.1/openvino_docs_ops_movement_Unique_10.html